var/spack/repos/builtin/packages/opencv/package.py

```diff
@@ -58,7 +58,7 @@ class Opencv(CMakePackage, CudaPackage):
     components = [
         '1394', 'ade', 'aravis', 'arith_dec', 'arith_enc', 'avfoundation', 'clp',
         'cuda', 'eigen', 'ffmpeg', 'freetype', 'gdal', 'gdcm', 'gphoto2', 'gstreamer',
-        'halide', 'hpx', 'imgcodec_hdr', 'imgcode_pfm', 'imgcodec_pxm',
+        'gtk', 'halide', 'hpx', 'imgcodec_hdr', 'imgcode_pfm', 'imgcodec_pxm',
```

Comment:
Shouldn't this be tied into adding another dependency when the variant is enabled?

Reply:
Yes, this needs to add a `gtkplus` dependency too. Also, it seems like this component is entirely hidden on macOS, so we may need to add a conflict for macOS.

Reply:
@bvanessen @adamjstewart incorporated the requested changes. Please check.